dist/index.js

```diff
@@ -41338,28 +41319,21 @@ const run = async () => {
   return;
 };
 
-function createContext(envFilters, ignoreKey = '') {
+function createContext(contextKey, filter, ignoreKey = '') {
   const ctx = {};
+
   Object.keys(process.env)
-    .filter(function (key) {
-      return process.env[key] != '';
-    })
-    .filter(function (key) {
-      if (key === ignoreKey) {
-        return false;
-      }
-      return true;
-    })
-    .forEach(function (key) {
-      envFilters.forEach(function (p) {
-        if (key.startsWith(p.prefix)) {
```

Comment:
This check was moved into a filter to make all of that a little cleaner to read.